src/coreclr/vm/dllimportcallback.cpp

```diff
@@ -1098,6 +1101,10 @@ PCODE TheUMEntryPrestubWorker(UMEntryThunk * pUMEntryThunk)
     if (pThread->IsAbortRequested())
         pThread->HandleThreadAbort();
 
+#if defined(HOST_OSX) && defined(HOST_ARM64)
+        auto jitWriteEnableHolder = PAL_JITWriteEnable(true);
```

Comment:
A nit - wrong indentation

Reply:
Someone should really fix cut and paste.  The number of times the clipboard created errors in the last decades. 😉